synfig-studio/src/gui/trees/historytreestore.cpp

```diff
@@ -41,6 +41,8 @@
 #include <synfig/general.h>
 #include <synfig/valuenode.h>
 
+#include <synfigapp/action_system.h>
```

Comment:
It looks like this header is not needed anymore too.

Reply:
oh yes definetly, because we are not using the flag anymore here.